packages/inline-video/style.module.css

```diff
@@ -0,0 +1,79 @@
+.inlineVideo {
+  --gradient-size: 32px;
+}
+
+.videoContainer {
+  --gradient-stops: #0654ef 0%, #cde6fe 100%;
+
+  width: 100%;
+  position: relative;
+
+  @media (--medium-up) {
+    @nest .left & {
+      padding-left: var(--gradient-size);
+    }
+
+    @nest .right & {
+      padding-right: var(--gradient-size);
+    }
+
+    &::before {
```

Comment:
Thoughts on just making this gradient the full width of the element and changing the padding left/right based on placement?

Reply:
Ah, I think I know what you mean! I'll give it a go and see how that looks

Reply:
It ended up working well! The changes are captured in the latest commit (along with a fix for a regression 😅)